catalog/transformer/catalog-transformer-bootflag/pom.xml

```diff
@@ -56,6 +56,32 @@
             <artifactId>bootflag-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
```

Comment:
✏️ Not an issue with this PR, but this bundle should be added to the `catalog-transformer-bootflag` feature in `catalog-app`.